tests/unit/functions/edh-marshaller.test.ts

```diff
@@ -71,20 +182,21 @@ describe('edhMarshaller Function', () => {
         jest.spyOn(console, 'log');
 
         try {
```

Comment:
There are nicer ways to test exceptions being thrown
`expect(t).toThrow("Uh oh!");`